### PR TITLE
Fix: Wrong Display saying Charges not found even when charges are pre…

### LINKED ISSUE #1122: Text and image saying charges not found even if charges are present in Client Charges layout
**Summary:** 
Displays messages "No charges to display" while simultaneously displaying charges

**Steps to reproduce:** 
Go to a particular client's charges page where a few charges are already present and you will observe the issue.

**Expected behaviour:** 
Text and image which says charges not found should not be displayed

**Observed behaviour:** 
Text and image which says charges not found is displayed which might cause a common user to get confused, and in general is not correct information being provided to the user.

**Device and Android version:** 
OnePlus 3T, Android 8.0, Stock ROM

**Screenshots:** 
Screenshot (11-Mar-2019 12_23_54 PM)

### PR DESCRIPTION
**Pull request by rachittt**
Fixes #1122 

Screenshot:
WhatsApp Image 2019-12-09 at 19 17 20

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by elirehema** [comment]
Hello @rachittt do not include any business logics in view i.e Fragment or Activity. View is not responsible for any business logic but Presenter is,  though is seems working.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientcharge/ClientChargeFragment.java [hunk 1]
```diff
@@ -247,11 +247,20 @@ public void onChargeCreatedFailure(String errorMessage) {
         Toaster.show(rootView, errorMessage);
     }
 
+    private int recyclerCount() {
```
Comment (laxyapahuja) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientcharge/ClientChargeFragment.java]: i don't think it's necessary to create another function just to get the item count of ```mChargesNameListAdapter```.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientcharge/ClientChargeFragment.java [hunk 2]
```diff
@@ -247,11 +247,20 @@ public void onChargeCreatedFailure(String errorMessage) {
...
+            count = mChargesNameListAdapter.getItemCount();
+        }
+        return count;
+    }
     @Override
     public void showEmptyCharges() {
-        ll_error.setVisibility(View.VISIBLE);
-        mNoChargesText.setText(getResources().getString(R.string.message_no_charges_available));
-        mNoChargesIcon.setImageResource(R.drawable.ic_assignment_turned_in_black_24dp);
+        if (recyclerCount() == 0) {
```
Comment (laxyapahuja) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientcharge/ClientChargeFragment.java]: could just be ```if (mChargesNameListAdapter.getItemCount() == 0) {```.

Comment (rachittt) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientcharge/ClientChargeFragment.java]: I tried that, but it was not correct as when you go to the client with no charges the message doesn't appear.

**Comment by laxyapahuja** [comment]
@rachittt looks good now. try not to make separate functions when not needed. :D

**Pull request merged**